sass/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "sass": "1.3.2"
```

Comment:
we should have doc somewhere for maintainers to know how to cut releases

Reply:
Let's do that in a follow-up PR, mainly because I don't know how to do that myself